index.html

```diff
@@ -168,7 +172,46 @@ <h3>Skills</h3>
                 </ul>
             </li>
         </ul>
+        </div>
+<div class="form-container">
+    <h4 class="aboutme-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+    <form action="https://formspree.io/f/xrgvbqgv" method="POST">
+        <input type="text" name="fullname" placeholder="Full Name" maxlength="30" required>
+        <input type="email" name="email" placeholder="Email Address" required>
+        <textarea type="text" name="textarea" maxlength="500" required>Enter text here</textarea>
```

Comment:
- [ ] Good job with the design. Currently, the text `Enter text here` is behaving like a preloaded value instead of a placeholder. I suggest you remove it and add it as a `placeholder` attribute on the `<textarea>` element so it can disappear when someone starts typing in it. `placeholder="Enter text here"` 🤝️